benchmarks/bench.sh

```diff
@@ -401,9 +405,14 @@ data_clickbench_1() {
     else
         URL="https://datasets.clickhouse.com/hits_compatible/hits.parquet"
         echo -n "... downloading ${URL} (14GB) ... "
-        wget --continue ${URL}
-    fi
-    echo " Done"
+        if ! wget --continue ${URL}; then
```

Comment:
I think the check above tests for the file size and already detects partial / failed previous downloads
```shell
    if test "${OUTPUT_SIZE}" = "14779976446"; then
```

So I am not sure this is necessary